packages/aws-cdk-lib/aws-route53/lib/record-set.ts

```diff
@@ -270,6 +276,12 @@ export class RecordSet extends Resource implements IRecordSet {
       aliasTarget: props.target.aliasTarget && props.target.aliasTarget.bind(this, props.zone),
       ttl,
       comment: props.comment,
+      geoLocation: props.geoLocation ? {
+        continentCode: props.geoLocation.continentCode,
+        countryCode: props.geoLocation.countryCode,
+        subdivisionCode: props.geoLocation.subdivisionCode,
+      } : undefined,
+      setIdentifier: props.geoLocation ? this.configureSetIdentifer(props.geoLocation) : undefined,
```

Comment:
Before merging this PR: I will check feature request #4391 to see if the logic for `setIdentifer` conflicts with other features that rely on the same field. I will give you feedback in the next few days. Please also feel free to think about it.

Reply:
Was this looked into or are we still awaiting feedback?